sdk/eventhub/Azure.Messaging.EventHubs/tests/EventHubClient/EventHubClientLiveTests.cs

```diff
@@ -278,5 +280,36 @@ await using (var client = new EventHubClient(connectionString))
                 }
             }
         }
+
+#if FullNetFx
```

Comment:
Were you able to test this and prove that it doesn't work when run for .NET core?    If so, I think that we should consider moving platform-specific stuff to a new file.

Reply:
I haven't tried it, but I can do something about it.

Reply:
In fact, it does work for Core. I have removed the preprocessor directives.